estate/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Real Estate',
```

Comment:
```suggestion
    'name': "Real Estate",
```
Strings should be in `""` and technical terms in `''`.